incubator/hnc/internal/reconcilers/test_helpers_test.go

```diff
@@ -18,17 +18,17 @@ import (
 	"sigs.k8s.io/multi-tenancy/incubator/hnc/internal/config"
 )
 
-// GVKs maps a kind to its corresponding GVK.
+// GVKs maps a resource to its corresponding GVK.
 var GVKs = map[string]schema.GroupVersionKind{
-	"Secret":        {Group: "", Version: "v1", Kind: "Secret"},
-	"Role":          {Group: "rbac.authorization.k8s.io", Version: "v1", Kind: "Role"},
-	"RoleBinding":   {Group: "rbac.authorization.k8s.io", Version: "v1", Kind: "RoleBinding"},
-	"NetworkPolicy": {Group: "networking.k8s.io", Version: "v1", Kind: "NetworkPolicy"},
-	"ResourceQuota": {Group: "", Version: "v1", Kind: "ResourceQuota"},
-	"LimitRange":    {Group: "", Version: "v1", Kind: "LimitRange"},
-	"ConfigMap":     {Group: "", Version: "v1", Kind: "ConfigMap"},
-	// CronTab is a custom resource.
-	"CronTab": {Group: "stable.example.com", Version: "v1", Kind: "CronTab"},
+	"secrets":               {Group: "", Version: "v1", Kind: "Secret"},
+	api.RoleResource:        {Group: api.RBACGroup, Version: "v1", Kind: api.RoleKind},
+	api.RoleBindingResource: {Group: api.RBACGroup, Version: "v1", Kind: api.RoleBindingKind},
```

Comment:
For future reference - I don't think we needed to make the change to the callers of these functions in this PR - leaving the _existing_ tests with "RoleBinding" when calling `makeObject` and `hasObject` would have made this PR quite a bit smaller without breaking functionality, and then you could have added it in a followup PR. But I don't think it's worth _undoing_ that change either.

Reply:
Thanks for the suggestion, agreed. I will resolve this without any changes.